libs/vkd3d/resource.c

```diff
@@ -3799,6 +3844,14 @@ static HRESULT d3d12_descriptor_heap_init(struct d3d12_descriptor_heap *descript
         }
     }
 
+    if (desc->Type == D3D12_DESCRIPTOR_HEAP_TYPE_CBV_SRV_UAV &&
```

Comment:
If the heap is not shader visible, can't we just use a plain vkd3d_malloc instead of mapping a Vulkan buffer?

Reply:
Actually, good point. We never read from that buffer anyway since the counter address is stored with the view, so I guess the proper solution would be to not create it in the first place.